opencog/nlp/relex2logic/rule-helpers.scm

```diff
@@ -466,26 +466,26 @@
 	(define adv (cog-name adv-node))
 	(define adv_instance (cog-name adv-instance-node))
 	(ListLink
-		(InheritanceLink  (ConceptNode adv_instance) (ConceptNode adv))
-			(ImplicationLink  (PredicateNode instance) (PredicateNode verb))
-			(r2l-wordinst-Predicate instance)
-			(r2l-wordinst-concept adv_instance)
-			(InheritanceLink  (SatisfyingSetLink (PredicateNode instance)) (ConceptNode adv_instance))
+		(ImplicationLink  (PredicateNode adv_instance) (PredicateNode adv))
+		(InheritanceLink  (ConceptNode instance) (ConceptNode verb))
```

Comment:
Since when are verbs "Concepts"?  I thought only nouns are "concepts", and verbs are "predicates", right? 

Or maybe the R2L subsystem doesn't actually make this kind of distinction?  Or maybe this distinction doesn't matter?